tests/gpu-tests/test_train.py

```diff
@@ -146,14 +182,45 @@ def test_dpo():
 
 
 @pytest.mark.gpu
-def test_rm():
+@pytest.mark.parametrize("test_mode", ["unit", "integration"])
+def test_rm(test_mode):
     model_path = os.getenv('NEMO_SKILLS_TEST_NEMO_MODEL')
     if not model_path:
         pytest.skip("Define NEMO_SKILLS_TEST_NEMO_MODEL to run this test")
     model_type = os.getenv('NEMO_SKILLS_TEST_MODEL_TYPE')
     if not model_type:
         pytest.skip("Define NEMO_SKILLS_TEST_MODEL_TYPE to run this test")
 
+    if test_mode == "unit":
+        input_dir_seeds = "/nemo_run/code/tests/data/score_rm_inputs"
+        input_dir_greedy = "/nemo_run/code/tests/data/score_rm_inputs"
+        expected_scores_per_file = 5
+    else:  # test_mode == "integration"
+        input_dir_greedy = os.getenv('NEMO_SKILLS_TEST_RM_INPUTS_GREEDY')
+        if not input_dir_greedy:
+            pytest.skip("Define NEMO_SKILLS_TEST_RM_INPUTS_SEEDS_GREEDY to run this test")
+        input_dir_seeds = os.getenv('NEMO_SKILLS_TEST_RM_INPUTS_SEEDS')
+        if not input_dir_seeds:
+            pytest.skip("Define NEMO_SKILLS_TEST_RM_INPUTS_SEEDS to run this test")
+        expected_scores_per_file = int(os.getenv('NEMO_SKILLS_TEST_RM_EXPECTED_SCORES_PER_FILE'))
+        if not expected_scores_per_file:
+            pytest.skip("Define NEMO_SKILLS_TEST_RM_EXPECTED_SCORES_PER_FILE to run this test")
+
+    test_config_path = Path(__file__).absolute().parent / "test-local.yaml"
+    config = yaml.safe_load(open(test_config_path).read())
+    volumes = config['mounts']
+    container = config['containers']['nemo-skills']
+    docker_run(
+        image_name=container,
+        volume_paths=volumes,
+        command=f'rm -rf /tmp/nemo-skills-tests/{model_type}/test-rm/{{training,score,model-averaged-nemo}}',
+    )
+    docker_run(
```

Comment:
This is likely unnecessary, the generate script should create the folder for you (if it doesn't, we need to change that)

Reply:
good catch--this was a holdover from when I was manually managing the directory permissions outside of docker. Works without it now--removed in 332a7cddf963afd1a4821f8d8e6932a311258b59